### PR TITLE
feat(cloud.project.serving): add serving engine endpoints

### PR DESCRIPTION
**Pull request by XciD**
Adding Serving Engine Endpoints for the new Lab

### CODE REVIEW

File: src/api/cloud/project/serving/namespaces/cloud-project-io-stream.v6.service.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectServingNamespaceV6', ($resource) => $resource('/cloud/project/:serviceName/ai/serving/:namespaceId', {
```
Comment (marie-j) [inline src/api/cloud/project/serving/namespaces/cloud-project-io-stream.v6.service.js]: When creating a route we should try to follow the API route . 
Here the path is `/cloud/project/:serviceName/ai/serving/:namespaceId` so ideally your service should be named `OvhApiCloudProjectAiServing`
`namespaceId` is a parameter so you would use a `.get()` for `/cloud/project/:serviceName/ai/serving/:namespaceId`  but with a `.query()` the path would be `/cloud/project/:serviceName/ai/serving`(no `Namespace` in the path)

Comment (XciD) [reply src/api/cloud/project/serving/namespaces/cloud-project-io-stream.v6.service.js]: reworked here 8b8c1fd9b2505b77313fb3e7b00f0336c73513ad

File: src/api/cloud/project/serving/capabilities/flavor/cloud-project-serving-capabilities-stream-region.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectAiServingCapabilitiesFlavorV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/ai/serving/:namespaceId/capabilities/flavor', {
+      serviceName: '@serviceName',
```
Comment (marie-j) [inline src/api/cloud/project/serving/capabilities/flavor/cloud-project-serving-capabilities-stream-region.v6.service.js]:
```suggestion
      serviceName: '@serviceName',
      namespaceId: '@namespaceId'
```

Comment (XciD) [reply src/api/cloud/project/serving/capabilities/flavor/cloud-project-serving-capabilities-stream-region.v6.service.js]: nice catch d303ff4e73508c01e78f2eb77315e63fd309265d

**Pull request merged**